sparsecoding/data/datasets/common.py

```diff
@@ -0,0 +1 @@
+RAW_DATA_DIR = "raw_data/"
```

Comment:
I like this idea; we should think about where to put this directory. I think this (combined with the OlshausenDataset class loading scheme) will make a new directory from wherever the user's present working directory is when it's called. 
I think this is what pytorch does with MNIST as well. 

Is there a better way to do this? Maybe this is fine.

Reply:
Oh yes, that's a good point. I will change to 
```
import sparsecoding
dir = sparsecoding.__file__
```
or something like that.

Reply:
done!